src/main/java/ru/sbt/mipt/oop/HomeEventProcessor.java

```diff
@@ -0,0 +1,44 @@
+package ru.sbt.mipt.oop;
+
+import static ru.sbt.mipt.oop.SensorEventType.*;
+import static ru.sbt.mipt.oop.SensorEventType.DOOR_CLOSED;
+
+public class HomeEventProcessor implements EventProcessor{
+    private final EventCreator eventCreator;
+    private final Logger logger;
+    private final LightEventProcessor lightEventProcessor;
+    private final DoorEventProcessor doorEventProcessor;
+    private Event event;
+    private SmartHomeImpl smartHome;
+
+    public HomeEventProcessor(Logger logger) {
+        this.eventCreator = new EventCreatorImpl();
+        this.logger = logger;
+        this.lightEventProcessor = new LightEventProcessor(logger);
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D0%B2%D0%B0%D0%B9%D1%82%D0%B5-%D0%B7%D0%B0%D0%B2%D0%B8%D1%81%D0%B8%D0%BC%D0%BE%D1%81%D1%82%D0%B8-%D1%81%D0%BD%D0%B0%D1%80%D1%83%D0%B6%D0%B8